src/environments/environment.ts

```diff
@@ -10,7 +10,7 @@ export const environment: Config = {
     product: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
     order: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
     import: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
-    bff: 'https://.execute-api.eu-west-1.amazonaws.com/dev',
+    bff: 'https://uzaf313c60.execute-api.us-east-1.amazonaws.com/dev',
```

Comment:
Great, but It's not enough for integration between your frontend and backend. As a proof your frontend just take mock data from frontend application.
![image](https://user-images.githubusercontent.com/73526360/222134425-dcea6e2a-cfe1-4c2e-939c-26ba305f3234.png)
As a result your cloudfrond app has to make request and reicive from your backend application data. In your case 
![image](https://user-images.githubusercontent.com/73526360/222136364-12b8f9fb-ece2-4280-828f-c29ac29cc11b.png)
cars should be displayed. 

Also the question about swagger, are going to add it, or no??